docs/source/guides/logical_types_and_semantic_tags.ipynb

```diff
@@ -676,6 +676,51 @@
     "\n",
     "In this way, a `ColumnSchema` can define a type space under which columns in a Woodwork DataFrame can fall."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "sexual-adoption",
+   "metadata": {},
+   "source": [
+    "## Checking for nullable logical types\n",
+    "\n",
+    "Some logical types support having null values in the underlying data while others do not. The logical types that do support null values are listed below:\n",
```

Comment:
Just wondering if we should list this separately here to document which logical types support null values, or should we include this information in the discussion of each individual logical type that happens earlier in the document? Having the list here means users would have to look in two different places to find all the info about a logical type. Also, if we left this list, maybe it would be better to list the types that do not support null values as that would be significantly shorter.

We could leave the cell demonstrating how to access the property, but not sure this long list is necessary?

Reply:
By putting the nullable property on the column accessor, I'd argue that we're specifically avoiding associating nullability with Logical Types. We're saying that nullability is only a relevant concept for a Woodwork column (series + column schema) and not any one of the pieces that make one up (logical type, physical type, etc).

It just so happens that there _is_ a relationship between nullability and logical type, but I think it's misleading to only explain `series.ww.nullable` in the concept of Logical Types when that's not how users will use it. 

The use case for telling users which logical types are nullable could be when a user is picking which logical types they use at init based on which are nullable or not. If that's the main use case, having nullability exist on an already created woodwork column is not useful for that purpose. 

The implementation here is useful for determining what you can do with your data once you know whether its nullable or not. For example: you'd check a column's nullability when deciding whether you need preprocessing to impute nulls. But for that, a user wouldn't need to know the logical type (the logical type is important for determining how to impute, but not whether to impute).

So I don't know that this concept of a column's nullability needs to be explained in the context of logical types. What makes sense to me as a way of explaining nullability in Woodwork with the implementation here is to focus less on how one determines whether a column is nullable, but what you do with that information. 

So maybe the docs entry doesn't belong in this guide, since it might look something like 

> A column in woodwork may contain null values. This can have an impact on how the column is used. Instead of checking whether a column has any null values, the `nullable` property of a woodwork column can be used to check if a column can contain null values in the first place. 
... example showing series.ww.nulalble usage ...

Reply:
Hmm...I just did a couple things.  I did add nullability information to each type's description in the docs.  If we go with what @tamargrey is saying and try to de-emphasize the relationship between logical types and nullability (which I'm in favor of doing), I can take all those individual nullability flags out.

I also updated the description of how nullability is determined to more transparently indicate the importance of the underlying pandas dtype.  There's no longer an exhaustive list of the nullable logical types but a description of how nullability is determined.

Reply:
@thehomebrewnerd I also updated the API docs to include the `WoodworkColumnAccessor.nullable` property.